qutebrowser/config/configdata.yml

```diff
@@ -370,6 +370,23 @@ content.default_encoding:
     The encoding must be a string describing an encoding such as _utf-8_,
     _iso-8859-1_, etc.
 
+content.unknown_url_scheme_policy:
+  type:
+    name: String
+    valid_values:
+      - DisallowUnknownUrlSchemes: Disallows all navigation requests to URLs with unknown schemes.
+      - AllowUnknownUrlSchemesFromUserInteraction: Allows navigation requests to URLs with unknown schemes that are issued from user-interaction (like a mouse-click), whereas other navigation requests (for example from JavaScript) are suppressed.
+      - AllowAllUnknownUrlSchemes: Allows all navigation requests to URLs with unknown schemes.
```

Comment:
Please use `disallow-unknown-url-schemes` etc. for consistency with other options.